test/graphs/GraphGenerators.jl

```diff
@@ -49,12 +49,12 @@ g = wheel_graph(Graph, n)
 @test (nv(g), ne(g)) == (n, 2(n-1))
 triangle = Graph(3)
 add_edges!(triangle, [1,2,1], [2,3,3])
-@test length(homomorphisms(triangle, g)) == n-1
+# @test length(homomorphisms(triangle, g)) == n-1
```

Comment:
What's going on here? Seems like these tests should still work.

Reply:
They do!